pkg/controller/nodes/executor.go

```diff
@@ -190,6 +190,7 @@ func (c *nodeExecutor) attemptRecovery(ctx context.Context, nCtx handler.NodeExe
 	case core.NodeExecution_SKIPPED:
 		return handler.PhaseInfoSkip(nil, "node execution recovery indicated original node was skipped"), nil
 	case core.NodeExecution_SUCCEEDED:
```

Comment:
should add a `fallthrough` here right? in this case it will only print the debug if status is RECOVERED i believe.

Reply:
Already out the door..
but I thought this would fall through just like in C. Sorry not fluent in go.. 😉

Reply:
But yeah.. Makes sense that an explicit fallthrough is needed, otherwise it would not have worked for succeeded either..
feel free to fix it, otherwise I will on Monday

Reply:
thanks for catching that, fixed.